recipes-core/packagegroups/packagegroup-ni-base.bb

```diff
@@ -83,7 +83,6 @@ RDEPENDS:${PN} += "\
 	openssh-sshd \
 	openvpn \
 	opkg \
-	opkg-keyrings \
```

Comment:
Wouldn't removing it from here cause BSI and safemode to not have this installed anymore? I don't think we'd want that.
I think it's still ok to have the package in both dist feed and core feed and if there is a newer version of package in dist-feed, opkg upgrade should pick up that version - but you should test this.

Reply:
> Wouldn't removing it from here cause BSI and safemode to not have this installed anymore? I don't think we'd want that.

Yes, you are right. It makes sense to not remove this from here. I will abandon this PR.

> I think it's still ok to have the package in both dist feed and core feed and if there is a newer version of package in dist-feed, opkg upgrade should pick up that version - but you should test this.

Got it. I can test this functionality once the other PR gets merged. Hope this is fine?

Reply:
You can test this without waiting on the other PR. Just host a local feed with newer version of keyrings package (i.e., make changes to keyring package and build it), add it to a target and do `opkg update; opkg upgrade` and verify that keyrings package gets upgraded.